### PR TITLE
[RTM] Use the Symfony session instead of $this->Session

### PR DESCRIPTION
**Pull request by leofeyer**
This PR replaces `$this->Session` with the Symfony session. It also attempts to store non-state data in the default session bag, which will not be persisted in the database.

### CODE REVIEW

File: src/Resources/contao/widgets/PageSelector.php [hunk 1]
```diff
@@ -139,14 +149,17 @@ public function generate()
 		}
 		else
 		{
-			$strNode = $this->Session->get('tl_page_picker');
+            /** @var AttributeBagInterface $objSessionBag */
```
Comment (Toflar) [inline src/Resources/contao/widgets/PageSelector.php]: CS

**Comment by leofeyer** [comment]
I have changed the status to RFC.

**Comment by Toflar** [comment]
:+1:

**Comment by leofeyer** [comment]
I have moved the referer logic from the `UserSessionListener` into a `StoreRefererListener` class. Please review again :)

**Comment by leofeyer** [comment]
I have changed the status to RTM.

**Pull request merged**